src/amber/router/pipe/csrf.cr

```diff
@@ -73,7 +73,9 @@ module Amber
             session_token = Base64.decode(real_session_token(context))
             return Crypto::Subtle.constant_time_compare(unmasked, session_token)
           end
-          return false
+          false
```

Comment:
What about making this a guard clause

```
return false unless request_token(context) && real_session_token(context)
```

Reply:
I feel like multiple guard clauses in a row are harder to follow, but I'm fine with that.